unit-tests/run-unit-tests.py

```diff
@@ -4,22 +4,21 @@
 # Copyright(c) 2020 Intel Corporation. All Rights Reserved.
 
 import sys, os, subprocess, locale, re, platform, getopt
+from abc import ABC
```

Comment:
?

Reply:
that's the only way to create an abstract class in python. I can remove it and make Test not abstract

Reply:
Huh... I never knew about ABC.. OK, but maybe add `# Abstract Base Class` for us ignorant people?